scala-lightautoml-transformers/src/main/scala/org/apache/spark/ml/feature/lightautoml/IsHolidayTransformer.scala

```diff
@@ -0,0 +1,164 @@
+package org.apache.spark.ml.feature.lightautoml
+
+import org.apache.hadoop.fs.Path
+import org.apache.spark.SparkException
+import org.apache.spark.ml.Transformer
+import org.apache.spark.ml.attribute.NumericAttribute
+import org.apache.spark.ml.feature.lightautoml.IsHolidayTransformer.IsHolidayTransformerWriter
+import org.apache.spark.ml.param.ParamMap
+import org.apache.spark.ml.param.shared.{HasInputCols, HasOutputCols}
+import org.apache.spark.ml.util._
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{DataFrame, Dataset, SparkSession}
+import org.apache.spark.util.VersionUtils.majorMinorVersion
+
+import scala.collection.JavaConverters._
+import scala.collection.{Map, mutable}
+
+class IsHolidayTransformer(override val uid: String, private var holidays_dates: Map[String, Set[String]])
+        extends Transformer
+                with HasInputCols
+                with HasOutputCols
+                with MLWritable {
+
+  val dt_format = "yyyy-MM-dd"
```

Comment:
could be in companion object, like [this](https://stackoverflow.com/questions/803206/use-class-variables-as-constants-in-scala)

Reply:
fixed